### PR TITLE
DX-569 Extension of Accounts EP with new CDR Fields

### PR DESCRIPTION
**Pull request by basiq-ash**
- Excluding Purses & Features
- Meta.fees.discounts added
- Meta.depositRates added
- Meta.lendingRates
- Meta.termDeposit
- Meta.CreditCard
- Meta.Loan
- Meta.Addresses
- Meta.Links

### CODE REVIEW

File: data.yml [hunk 1]
```diff
@@ -1707,21 +1686,29 @@ components:
...
       - accountOwnership
-      - name
-      - currency
+      - bundleName
       - balance
       - availableFunds
-      - lastUpdated
-      - institution
       - creditLimit
+      - amortisedLimit
```
Comment (basiq-chris) [inline data.yml:1704]: Maybe next sprint, have something that specifies what this is.

Comment (basiq-ash) [reply data.yml:1704]: @basiq-chris its defined in the description and also taken from [here](https://basiq.atlassian.net/wiki/spaces/COM/pages/1892614434/Open+Banking+Account+Data~text=Object%20representing%20the%20available%20limit%20amortised%20according%20to%20payment%20schedule.%20Assumed%20to%20be%20zero%20if%20absent).

Comment (basiq-ash) [reply data.yml:1704]: defined [here](https://github.com/basiqio/api-ref/pull/331/files#diff-093f056f1d238f3fb59742e58ef57a2a0d056e07e42e18a72b451da2801a489eR1763). @basiq-chris

File: data.yml [hunk 2]
```diff
@@ -2030,7 +2408,7 @@ components:
           items:
             $ref: '#/components/schemas/AccountResponseResource'
         links:
-          $ref: '#/components/schemas/ResourceLink'
+          $ref: '#/components/schemas/ResourceaccountLink'
```
Comment (basiq-chris) [inline data.yml:2411]: Should probably capitalise that A in `account`

**Review by basiq-chris — APPROVED** [review]
Looks good, just requesting semantic changes

File: data.yml (hunk 1 above)
Comment (basiq-chris) [reply data.yml:1704]: Meant more what the word `amortised` meant

File: data.yml (hunk 2 above)
Comment (basiq-ash) [reply data.yml]: @basiq-chris Nice pickup, here you go, i have fixed it [here](https://github.com/basiqio/api-ref/pull/331/commits/45b5a5ea774036f2d48e9742aff5f3d79a19abb0).

File: data.yml [hunk 3]
```diff
@@ -1843,32 +1855,93 @@ components:
...
-              description: >-
-                (optional) - Minimum amount of next instalment.
-              example: "5500"
+              description: Minimum amount of next instalment. 
+            minInstalmentCurrency:
+              type: string
+              description: If absent assumes AUD. 
+            maxRedraw: 
+              type: string
+              description: Maximum amount of funds that can be redrawn. If not present redraw is not available even if the feature exists for the account.
```
Comment (basiq-gabriel) [inline data.yml:1923]: Add a comma: "If not present, redraw is not available..."

File: data.yml [hunk 4]
```diff
@@ -1843,32 +1855,93 @@ components:
...
+          properties: 
+            lodgementDate:
+              type: string
+              description: The lodgement date of the original deposit. 
+            maturityDate:
+              type: string
+              description: Maturity date for the term deposit. 
+            maturityAmount: 
+              type: string
+              description: Amount to be paid upon maturity. If absent it implies the amount to paid is variable and cannot currently be calculated.
```
Comment (basiq-gabriel) [inline data.yml:1873]: Add a comma: "If absent, it implies the amount..."

File: data.yml (hunk 1 above)
Comment (basiq-ash) [reply data.yml:1704]: sure, will add in the next sprint.

File: data.yml (hunk 3 above)
Comment (basiq-ash) [reply data.yml]: Fixed.

File: data.yml (hunk 4 above)
Comment (basiq-ash) [reply data.yml]: Fixed.

**Review by basiq-gabriel — APPROVED** [review]

**Review by nikolaratkovic — APPROVED** [review]

**Pull request merged**